tools/ctl/ctl.c

```diff
@@ -68,24 +80,27 @@ static void usage(char *name)
 	fprintf(stdout, "\t %s -h\n", name);
 	fprintf(stdout, "\nWhere:\n");
 	fprintf(stdout, " -D device name (default is hw:0)\n");
-	fprintf(stdout, " -g <size> generates");
-	fprintf(stdout, " the current ABI header with given payload size\n");
 	fprintf(stdout, " -c control name e.g.");
 	fprintf(stdout, " numid=22,name=\\\"EQIIR1.0 EQIIR\\\"\"\n");
 	fprintf(stdout, " -n control id e.g. 22\n");
+	fprintf(stdout, " -I IPC type (0 - IPC3, 1 - IPC4), defaults to 0\n");
```

Comment:
I think xtensa-build-zephyr.py has a more intuitive convention for "-i" / "--ipc": "IPC4" to force selection

Reply:
We should really say ```-I 3``` for IPC3 and ```-I 4``` for IPC4